### DISCUSSION TITLE
RustDesk Server: Location of connection log files on Windows server

### DISCUSSION THREAD

**Discussion by zackholz** [rustdesk/rustdesk-server#470] [Q&A]
Concerning RustDesk Server (OSS, not Pro Version), I'd like to know: Does it store connection log files of all managed connections between RustDesk clients? And where would this location be?

I've successfully set up a windows server and 2 client computers with RustDesk client software. The server got successfully installed hbbs and hbbr service via NSSM. Both clients can establish RustDesk connections successfully in both directions, no problem so far.

Concerning the log files: I see them on the client computers located in C:\Users\...\AppData\Roaming\RustDesk\log - and they are correctly written, any time there is a connection.

But on the server, I don't find any log files of the managed connections. Neither in C:\Windows\temp, nor in C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk, nor in any subfolder of C:\Users or C:\ProgramData - I just found nothing.

Do log files on the server exist at all?

**Comment by hueldoeu** [comment]
can someone please help me setting up rustdesk server on windows servee?
